Gemfile.lock

```diff
@@ -132,7 +120,7 @@ DEPENDENCIES
   rspec (~> 3.5.0)
 
 RUBY VERSION
-   ruby 3.1.3p185
+   ruby 3.3.0p0
```

Comment:
I am actually unsure why this was left at 3.1.3 in https://github.com/GSA-TTS/identity-site/pull/1269 -- it just didn't get updated. 😕 This part seems like a good change.

Reply:
Agree that this is a good change! After looking at my history I understand what ended up happening - I unintentionally did an upgrade on the Gemfile.lock and I think with all of the other changes that got made to it, it's a bit over the top for this PR. I'm going to clean it up from this PR but happy to talk about making the change in a separate branch!